entry.go

```diff
@@ -110,6 +110,10 @@ func (entry *Entry) Debug(args ...interface{}) {
 	}
 }
 
+func (entry *Entry) debug(args ...interface{}) {
+	entry.log(DebugLevel, fmt.Sprint(args...))
+}
```

Comment:
Yeah. I could remove them and call the original (publicly exported) ones. What do you think?